force-app/main/default/classes/SOQL.cls

```diff
@@ -1196,13 +1222,19 @@ public virtual inherited sharing class SOQL implements Queryable {
             }
             return this;
         }
-
+        
```

Comment:
@salberski Great PR!! <3 Thank you! I am really happy about that change! 

One question:
Can we make it a little bit more generic? 

Like 
```java
private String wrapper = '{0}';

 public Filter notLike() {
    this.wrapper = 'NOT({0})';
    return this;
}

public override String toString() {
    if (skipBinding) {
          return String.format(wrapper, new List<String>{ field + ' ' + comperator + ' ' + value });
    }
    
    return String.format(wrapper, new List<String>{ field + ' ' + comperator + ' ' + binder.bind(value) });
}
```

Reply:
Great suggestion, I'll do this!

Reply:
Done